configs/4core ebike program.nrel-op.json

```
@@ -0,0 +1,103 @@
+{
+  "url_abbreviation": "4core-ebike-program",
```

```suggestion
  "url_abbreviation": "4core-ebike",
```
this is the name of the environment according to Jianli